types/index.d.ts

```diff
@@ -0,0 +1,377 @@
+// TypeScript Version: 2.4
+export default Long;
```

Comment:
Might be that we have to add something around [here](https://github.com/dcodeIO/long.js/blob/master/src/long.js#L1) to make this work with just the sources under node.

Reply:
Ah, yeah I had to use `esModuleInterop: true` in my `tsconfig.json` to make this work. It'll probably be a webpack setting I'm assuming.

Reply:
Issue is that the sources are served under node, so no webpack or tsc involved. Should probably manually add what tsc does with `esModuleInterop: true` to `src/index.js`.

Reply:
In that case I'm confused. How would this TypeScript declarations file be affecting Node?

Reply:
> I had to use `esModuleInterop: true` in my `tsconfig.json` to make this work.

Just to be clear, this was in reference to another TS project, not this one.

Reply:
I see. I thought more in a sense of "polyfilling" something into `src/long.js` that would allow a loader using `import default from "long";` to actually work (it wouldn't currently, I think). I haven't looked this up yet, but I think there might be a workaround like this in loaders if the exported object has an `__esModule` property plus a `default` property that indicates the default export - but I might as well be misunderstanding something here.